[2023-05-21T21:35:39.340+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_todays_gamesv3.scrape_prob_task manual__2023-05-21T21:35:27.852278+00:00 [queued]>
[2023-05-21T21:35:39.363+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_todays_gamesv3.scrape_prob_task manual__2023-05-21T21:35:27.852278+00:00 [queued]>
[2023-05-21T21:35:39.364+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-05-21T21:35:39.364+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 1
[2023-05-21T21:35:39.364+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-05-21T21:35:39.384+0000] {taskinstance.py:1303} INFO - Executing <Task(PythonOperator): scrape_prob_task> on 2023-05-21 21:35:27.852278+00:00
[2023-05-21T21:35:39.390+0000] {standard_task_runner.py:55} INFO - Started process 603 to run task
[2023-05-21T21:35:39.399+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_todays_gamesv3', 'scrape_prob_task', 'manual__2023-05-21T21:35:27.852278+00:00', '--job-id', '185', '--raw', '--subdir', 'DAGS_FOLDER/todays_gamesv3.py', '--cfg-path', '/tmp/tmp13bykokn']
[2023-05-21T21:35:39.399+0000] {standard_task_runner.py:83} INFO - Job 185: Subtask scrape_prob_task
[2023-05-21T21:35:39.511+0000] {task_command.py:388} INFO - Running <TaskInstance: get_todays_gamesv3.scrape_prob_task manual__2023-05-21T21:35:27.852278+00:00 [running]> on host b03c83f459ca
[2023-05-21T21:35:39.600+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=get_todays_gamesv3
AIRFLOW_CTX_TASK_ID=scrape_prob_task
AIRFLOW_CTX_EXECUTION_DATE=2023-05-21T21:35:27.852278+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-21T21:35:27.852278+00:00
[2023-05-21T21:35:39.602+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/functions/_scrape_prob.py", line 4, in _scrape_prob
    win_prob = scrapeProb('NYY', dt.datetime.now().year).scrape_prob()
NameError: name 'scrapeProb' is not defined
[2023-05-21T21:35:39.612+0000] {taskinstance.py:1326} INFO - Marking task as FAILED. dag_id=get_todays_gamesv3, task_id=scrape_prob_task, execution_date=20230521T213527, start_date=20230521T213539, end_date=20230521T213539
[2023-05-21T21:35:39.627+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 185 for task scrape_prob_task (name 'scrapeProb' is not defined; 603)
[2023-05-21T21:35:39.654+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-05-21T21:35:39.714+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
